tests/playwright/src/specs/volume-smoke.spec.ts

```diff
@@ -62,16 +68,84 @@ describe('Volume workflow verification', async () => {
 
-    const volumeRow = await volumesPage.getVolumeRowByName(volumeName);
+    const createVolumePage = await volumesPage.openCreateVolumePage(volumeName);
+    volumesPage = await createVolumePage.createVolume(volumeName);
+
+    await playExpect.poll(async () => await volumesPage.waitForVolumeExists(volumeName)).toBeTruthy();
+
+    //delete it from the details page
+    volumesPage = await navBar.openVolumes();
+    await playExpect(volumesPage.heading).toBeVisible();
+
+    let volumeRow = await volumesPage.getVolumeRowByName(volumeName);
     playExpect(volumeRow).not.toBeUndefined();
 
     const volumeDetails = await volumesPage.openVolumeDetails(volumeName);
     volumesPage = await volumeDetails.deleteVolume();
 
     await playExpect.poll(async () => await volumesPage.waitForVolumeDelete(volumeName)).toBeTruthy();
+    volumeRow = await volumesPage.getVolumeRowByName(volumeName);
+    playExpect(volumeRow).toBeUndefined();
+  });
+
+  test('Create volumes from bootc-image-builder', async () => {
+    //pull image from quay.io/centos-bootc/bootc-image-builder
+    const navigationBar = new NavigationBar(page);
+    let images = await navigationBar.openImages();
+    const pullImagePage = await images.openPullImage();
+    images = await pullImagePage.pullImage(imageToPull, imageTag);
+    await playExpect.poll(async () => await images.waitForImageExists(imageToPull)).toBeTruthy();
+
+    //count the number of existing volumes
+    let volumesPage = await navigationBar.openVolumes();
+    const previousVolumes = await volumesPage.countRowsFromTable();
+
+    //start a container from the image (generates 4 new volumes)
+    images = await navigationBar.openImages();
+    const imageDetails = await images.openImageDetails(imageToPull);
+    const runImage = await imageDetails.openRunImage();
+    let containers = await runImage.startContainer(containerToRun, containerStartParams);
+    await playExpect(containers.header).toBeVisible();
+    await playExpect
+      .poll(async () => await containers.containerExists(containerToRun), { timeout: 10000 })
+      .toBeTruthy();
+    await containers.startContainer(containerToRun);
+
+    //check that four volumes are created (in addition to the existing ones)
+    volumesPage = await navigationBar.openVolumes();
+    await playExpect(volumesPage.heading).toBeVisible();
+    const newVolumes = await volumesPage.countRowsFromTable();
+    playExpect(newVolumes - previousVolumes === 4).toBeTruthy();
+
+    //delete the container
+    containers = await navigationBar.openContainers();
+    const containersPage = await containers.deleteContainer(containerToRun);
+    await playExpect(containersPage.heading).toBeVisible();
+    await playExpect
+      .poll(async () => await containersPage.containerExists(containerToRun), { timeout: 15000 })
+      .toBeFalsy();
+
+    //prune unused volumes
+    volumesPage = await navigationBar.openVolumes();
+    await volumesPage.pruneVolumes();
```

Comment:
Shouldnt there be some validation here to check that the prune worked?

Reply:
I'm not sure how the check that the prune worked. I can compare to the previous number of volumes, but it's possible that some of the previous volumes were inactive and got pruned so it's not 100% accurate. Should I prune the volumes before starting the test?

Reply:
That's certainly one possibility, another would be to check that all the rows that are still there are for volumes that are still in use.

Choose whichever version you think is best.